Source/Classes/ActionHandling/Link.swift

```diff
@@ -46,12 +46,12 @@ extension NantesLabel {
 
     /// Adds a link to a `url` with a specified `range`
     @discardableResult
-    open func addLink(to url: URL, withRange range: NSRange) -> NantesLabel.Link? {
-        return addLinks(with: [.linkCheckingResult(range: range, url: url)], withAttributes: linkAttributes).first
+    open func addLink(to url: URL, withRange range: NSRange, linkTappedBlock: NantesLabel.LinkTappedBlock? = nil) -> NantesLabel.Link? {
+		return addLinks(with: [.linkCheckingResult(range: range, url: url)], withAttributes: linkAttributes, linkTappedBlock: linkTappedBlock).first
```

Comment:
@PatrickDotStar can you fix the indentation here to match the rest of the file?

Reply:
40c3b441a44e26da645eccbe2f9ff7ec7adbf9bc

Reply:
I will add unit tests if possible when I have some time :)